<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardListWindow</class>
 <widget class="QDialog" name="CardListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>1051</width>
       <height>43</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="PlayerInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Showing cards for Player X&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="jailCardLabel">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>10</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Out Of Jail cards owned:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sellJailCard">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>10</y>
        <width>131</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Sell Jail card</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="CardListScrollArea">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1049</width>
        <height>563</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>1049</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ExitButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
